pkg/cri/cri.go

```diff
@@ -0,0 +1,79 @@
+package cri
+
+import (
+	"context"
+	"errors"
+
+	"google.golang.org/grpc"
+	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+
+	log "github.com/cihub/seelog"
+)
+
+const (
+	criSocketPath = "unix:///var/run/cri.sock"
+)
+
+// ContainerInfo provides container information
+type ContainerInfo struct {
+	ID        string
+	Namespace string
+	Name      string
+	K8SUID    string
+}
+
+type APIs interface {
+	GetRunningContainers() (map[string]*ContainerInfo, error)
+}
+
+type Client struct{}
+
+func New() *Client {
+	return &Client{}
+}
+
+func (c *Client) GetRunningContainers() (map[string]*ContainerInfo, error) {
```

Comment:
nit: might be better to name this `GetRunningSandboxContainers()` to make it clearer at the calling point what this is actually doing. :)

Reply:
Good idea! For consistency with Kubernetes and the CRI (and thus less confusion hopefully), I think it's probably better to consistently refer to them as "pod sandboxes." The fact that a sandbox is a container under the hood is a common implementation choice but not universal: e.g. under Kata Containers a sandbox is a full-blown VM in which other containers run.

I'm renaming things now to use the pod sandbox terminology -- let me know if you'd prefer a different name.